.travis.yml

```diff
@@ -17,3 +17,4 @@ script:
   - (cd tests; pytest -s test-addons.py)
   - sudo microk8s.reset
   - sudo snap remove microk8s
+  - UPGRADE_MICROK8S_FROM=edge UPGRADE_MICROK8S_TO=`pwd`/*.snap pytest -s ./tests/test-upgrade.py
```

Comment:
Are we really starting with edge and upgrading to something else?

Reply:
Yes! Here is why. As soon as we merge a change the builders will trigger and push a new snap in to edge. So we need to make sure the snap we created (`UPGRADE_MICROK8S_TO=pwd/*.snap`) is backwards compatible and will not break what is already available on edge (`UPGRADE_MICROK8S_FROM=edge`).

Reply:
ok, so edge to freshly built not in the store snap. Makes sense to me, but the tests don't like *.snap, I think you need to get that resolved to filenames or something.